prisma/factories/plus_status.ts

```diff
@@ -0,0 +1,22 @@
+import { Factory } from "fishery";
+import { PlusStatus, PlusRegion } from "@prisma/client";
+import prisma from "../client";
+import userFactory from './user';
+import _ from "lodash";
```

Comment:
I was actually kind of surprised lodash is available since it's not a dependency. I guess we can still use dependencies of dependencies like this? Probably is fine but maybe adding it explicitly as devDependency is the best.

Reply:
Ah, yeah, this makes total sense – I didn't even check if was already included. I can add it as a devDependency, unless you think there's a better way to grab a random item from an array.

Reply:
Yeah I think I will write a custom helper for this. See if we can also make Typescript happy without type coercion that way. 

Generally not against using Lodash just have gotten the impression most of its claim to fame was from a time when JS was more limited and personally I haven't felt the need to use it.